### ISSUE TITLE
Add option to install Vernet datasets separately

### ISSUE THREAD

**Issue by KristinaRiemer** [weecology/retriever#819]
Currently when someone installs the Vertnet dataset, it automatically downloads all five datasets (Amphibian, Reptile, Mammal, Bird, and Fish). Because they're so large, it would be helpful to have the option to specify downloading only one of the five datasets at a time, especially because some people will only need/use one of them.

**Comment by zhangcandrew** [comment]
@henrykironde Do you think it'd be better if we made 5 different scripts instead of 1 with the option to choose from any of the 5 included?

**Comment by ethanwhite** [comment]
Seems like we have a couple of choices here:

1. Keep the current scripts and add additional scripts for each separately `vertnet-amphibians`, `vertnet-reptiles`...
2. Just have separate scripts for each separately `vertnet-amphibians`, `vertnet-reptiles`..., without an "everything script
3. Leave things they way they are

I don't see any harm to (1), except that we'll need to remember to update 6 scripts when there is a new release.

A bigger project would be to create a dataset type that is a combination of multiple datasets, so that `vertnet` just becomes a list of the 5 datasets that make it up. That would be a cool functionality to add at some point. but is probably a bit far off to be a solution to this (and it would require creating the separate files anyway).

What are folks thoughts on the 3 options above?

**Comment by henrykironde** [comment]
I will go with, Keep the current scripts and add additional scripts for each separately vertnet-amphibians, vertnet-reptiles...

If anyone wants to work on this, tag your self or comment.

**Comment by zhangcandrew** [comment]
I thinks that's a good way to handle the situation, since they will still have the option to download data sets from vertnet if they would like to. In the future, would it be worth it to add a feature that lets users download multiple datasets in one command? 

Something like: 
```
retriever install csv iris portal pantheria
```
to install csvs for the iris, portal, and pantheria datasets all at once?

**Comment by ethanwhite** [comment]
> In the future, would it be worth it to add a feature that lets users download multiple datasets in one command?

Yes, that would be great!

**Comment by ShivamNegi** [comment]
I would like to work on this. 
So 5 new scripts need to be made namely ```vertnet-amphibians```, ```vertnet-birds```, ```vertnet-fishes```, ```vertnet-mammals```, ```vertnet-reptiles```.
@zhangcandrew that would be a very useful addition.

**Comment by henrykironde** [comment]
Yes

**Comment by ShivamNegi** [comment]
How do I add a new script to Data Retriever list?
```
The dataset vertnet_amphibians isn't currently available in the Retriever
Run 'retriever ls to see a list of currently available datasets
```

**Comment by henrykironde** [comment]
You need to create a new script in the script folder. For this case you will copy the [vertnet.py](https://github.com/weecology/retriever/blob/master/scripts/vertnet.py),  and rename the copy with the new name say `vertnet_amphibians`  

In the file, remove variables that are not for vertnet_amphibians.
Do the same for  the others.

Whenever we create a new script or edit a script, we have to update `version.txt` by running `version.py`

**Comment by ShivamNegi** [comment]
I have created ```vertnet_amphibians.py``` under the ```scripts``` folder. But on running ```version.py```  there is no new entry for ```vertnet_amphibians.py``` getting created in ```version.txt```. Hence on running the command I get this error message
```
➜  retriever git:(vertnet-dataset) ✗ retriever install csv vertnet_amphibians
The dataset vertnet_amphibians isn't currently available in the Retriever
Run 'retriever ls to see a list of currently available datasets
```
I even ran ```retriever update```, to no effect. Could you please help.
I think the reason for this is that a ```.pyc``` is not getting generated by ```__init__.py``` for ```vertnet_amphibians.py```.

**Comment by henrykironde** [comment]
Create a PR and I have a look at it

**Comment by ethanwhite** [comment]
Implemented in 489d45b2f70f75a710d32adbd40363cad9e47b40. Thanks again @ShivamNegi!
